src/convertToObject.js

```diff
@@ -6,7 +6,22 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const arrayOfIndividualElements = sourceString.split(';');
+
+  const complexStylesObj = arrayOfIndividualElements
+    .map((el) => el.trim())
+    .filter((el) => el !== '')
```

Comment:
[CODE STYLE] - To improve readability, ensure that all chained methods are aligned with the same indentation. This helps to visually separate the chain of methods from the rest of the code.